### PR TITLE
cadaver: add page

### PR DESCRIPTION
**Pull request by Graywolf9**
A command-line WebDAV client for Unix.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Graywolf9** [comment]
Thank you so much for the review @sebastiaanspeck 

I was looking that `example.com` is a used domain name, eg:

https://github.com/tldr-pages/tldr/blob/8722bc8f1ceaeb69c979380b114b292dfe898725/pages/common/curl.md?plain=1#L13

Also, the part of the subdomain `mydomain` is an important one (maybe this one could be changed to `dav`)

I made some of the changes

File: pages/common/cadaver.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# cadaver
+
+> A command-line WebDAV client for Unix.
+> More information: <https://manned.org/cadaver>.
+
+- Connect to the server dav.example.com, open the root collection:
+
+`cadaver http://dav.example.com/`
```
Comment (gutjuri) [inline pages/common/cadaver.md:8]:
```suggestion
`cadaver {{http://dav.example.com/}}`
```

File: pages/common/cadaver.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Connect to the server dav.example.com use port 8022 and open the collection "/foo/bar/":
+
+`cadaver http://dav.example.com:8022/foo/bar/`
```
Comment (gutjuri) [inline pages/common/cadaver.md:12]:
```suggestion
`cadaver {{http://dav.example.com:8022/foo/bar/}}`
```

File: pages/common/cadaver.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Connect to a server called davs.example.com use SSL:
+
+`cadaver https://davs.example.com/`
```
Comment (gutjuri) [inline pages/common/cadaver.md:16]:
```suggestion
`cadaver {{https://davs.example.com/}}`
```

File: pages/common/cadaver.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Connect to the server dav.example.com, open the root collection:
+
+`cadaver http://dav.example.com/`
+
+- Connect to the server dav.example.com use port 8022 and open the collection "/foo/bar/":
+
+`cadaver http://dav.example.com:8022/foo/bar/`
+
+- Connect to a server called davs.example.com use SSL:
```
Comment (gutjuri) [inline pages/common/cadaver.md:14]:
```suggestion
- Connect to a server using SSL:
```

File: pages/common/cadaver.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# cadaver
+
+> A command-line WebDAV client for Unix.
+> More information: <https://manned.org/cadaver>.
+
+- Connect to the server dav.example.com, open the root collection:
+
+`cadaver http://dav.example.com/`
+
+- Connect to the server dav.example.com use port 8022 and open the collection "/foo/bar/":
```
Comment (gutjuri) [inline pages/common/cadaver.md:10]:
```suggestion
- Connect to a server using a specific port and open the collection "/foo/bar/":
```

**Review by gutjuri — COMMENTED** [review]
Indeed, I think that using `example.com` is fine. However, URLs in examples should use the `{{}}` placeholder syntax.

File: pages/common/cadaver.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# cadaver
+
+> A command-line WebDAV client for Unix.
```
Comment (kbdharun) [inline pages/common/cadaver.md:3]:
```suggestion
> WebDAV client for Unix.
```

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

File: pages/common/cadaver.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://manned.org/cadaver>.
+
+- Connect to the server dav.example.com, open the root collection:
```
Comment (kbdharun) [inline pages/common/cadaver.md:6]:
```suggestion
- Connect to the server <dav.example.com>, open the root collection:
```

We can enclose the link inside `<>` to make it clickable.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Comment by Graywolf9** [comment]
Thank you so much for your reviews @gutjuri and @kbdharun !

**Review by gutjuri — APPROVED** [review]

**Pull request merged**